testdata/orgnumber.json

```diff
@@ -0,0 +1,50 @@
+[
+    {
+        "integer": 8572090606,
+        "short_format": "8572090606",
+        "separated_format": "857209-0606",
+        "valid": false,
+        "type": "org",
+        "comment": "Valid Organisationsnummer (AP fond) - None valid Personnummer"
```

Comment:
Typo.

["invalid"](https://www.oxfordlearnersdictionaries.com/definition/english/invalid_1) or ["non-valid"](https://www.merriam-webster.com/dictionary/nonvalid) are valid spellings (no pun intended).

For example:
> None of the them are valid.
> The Personnummer is non-valid.
> Invalid Personnummer.

Reply:
Fixed in #40 ;P